### PR TITLE
Add dev notes on working with openshift-ansible.

### PR DESCRIPTION
**Pull request by dgoodwin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,40 @@ real openshift-ansible image to use in the ANSIBLE_IMAGE variable.
...
 
+## Developing With OpenShift Ansible
+
+You can build your own custom version of OpenShift Ansible when working on playbooks and roles from the openshift-ansible repo:
+
+  `docker build -f images/installer/Dockerfile -t dgoodwin-origin-ansible:latest .`
+
+You can then use this image with Cluster Operator via:
+
+  `USE_REAL_AWS=1 ANSIBLE_IMAGE="dgoodwin-origin-ansible:latest" ANSIBLE_IMAGE_PULL_POLICY="Never" contrib/examples/create-cluster.sh`
```
Comment (staebler) [inline README.md]: USE_REAL_AWS is not necessary and not used when ANSIBLE_IMAGE is used.

File: README.md [hunk 2]
```diff
@@ -49,3 +49,40 @@ real openshift-ansible image to use in the ANSIBLE_IMAGE variable.
 	* `USE_REAL_AWS=1 contrib/examples/create-cluster.sh`
 	* `ANSIBLE_IMAGE=openshift/origin-ansible:latest contrib/examples/create-cluster.sh`
 
+## Developing With OpenShift Ansible
+
+You can build your own custom version of OpenShift Ansible when working on playbooks and roles from the openshift-ansible repo:
```
Comment (abutcher) [inline README.md]: Could we link to the openshift-ansible repo here?

Comment (abutcher) [reply README.md]: Oh and could you fix the bulleted list at the bottom of the README while you're here :bowtie:

Comment (dgoodwin) [reply README.md]: Added the link, could you clarify what's wrong with a bulleted list, I'm not seeing anything.

Comment (abutcher) [reply README.md]: Lines 49/50 right above your changes are indented with tabs or something so they're just being shown inline rather than as a list.

**Comment by dgoodwin** [comment]
I gotcha now, updated.

**Pull request merged**